deploy/00_resolve_keep_registry.ts

```diff
@@ -11,10 +11,15 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   if (KeepRegistry && helpers.address.isValid(KeepRegistry.address)) {
     log(`using existing KeepRegistry contract at ${KeepRegistry.address}`)
   } else if (
-    hre.network.name !== "hardhat" ||
+    hre.network.name === "hardhat" &&
     (hre.network.config as HardhatNetworkConfig).forking.enabled
   ) {
     throw new Error("deployed KeepRegistry contract not found")
+  } else if (
+    hre.network.name !== "hardhat" &&
+    hre.network.name !== "development"
+  ) {
+    throw new Error("deployed KeepRegistry contract not found")
```

Comment:
We should try to combine these two `else if` statements.

Reply:
Maybe we could use optional chaining?
`(hre.network.config as HardhatNetworkConfig)?.forking?.enabled`

Reply:
Yea, I like it, I didn't know about this optional chaining. [7005ee1](https://github.com/threshold-network/solidity-contracts/pull/108/commits/7005ee1aed1192fc1c70fe94c2b2f59b4fb1231a)